pkg/adapter/apiserver/ref_test.go

```diff
@@ -11,36 +11,42 @@ func TestRefAddEvent(t *testing.T) {
 	d, ce := makeRefAndTestingClient()
 	d.Add(simplePod("unit", "test"))
 	validateSent(t, ce, sourcesv1alpha1.ApiServerSourceAddRefEventType)
+	validateMetric(t, d.reporter, 1)
```

Comment:
What would be even cooler is if you also validate the tags of the metric...

Reply:
agree!
Let me do that in a later PR, as we do not have access to the tags underneath... we will have to do some refactor for that...